src/cubi_tk/snappy/pull_sheets.py

```diff
@@ -63,7 +65,9 @@ def setup_argparse(parser: argparse.ArgumentParser) -> None:
     )
 
     parser.add_argument(
-        "--library-types", choices=("WES", "WGS", "Panel_seq"), help="Library type(s) to use, comma-separated, default is to use all."
+        "--library-types",
+        nargs= "*",  # same as default = []
+        help="Library type(s) to use( e.g. WES, WGS, or Panel_seq) paassed like '--library-types WES Panel_seq', default is to use all."
```

Comment:
This should keep the `choices` argument, handing _any_ kind of string to the isa downloader is going to cause problems otherwise